### PR TITLE
Added visualization utilities and example notebooks. Fixes JIRA AUTOCONFIG 68, 70

### PR DESCRIPTION
**Pull request by jcwbacker**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #iPython
 /.ipynb*
+/.ipynb*/
```
Comment (kree) [inline .gitignore]: It looks like this line was accidentally repeated?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
```
Comment (kree) [inline .gitignore]: It looks like this line was accidentally duplicated?

**Comment by kree** [comment]
Also, just to check, did you deliberately switch back to using from_adjacency, rather than from_adjacency_file ?

File: autocnet/utils/visualization.py [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    # create a subplot with its own axes
+    ax1 = fig.add_subplot(rows, columns, 1)
+    ax1.axis('off')
+    ax1.margins(tight=True)
+    plotFeatures(imageName1, 
+                 graph.get_keypoints(imageName1), 
+                 'ro', 
+                 featurePointSize)
+    ax2 = fig.add_subplot(rows, columns, 2)
+    ax1.axis('off')
```
Comment (jlaura) [inline autocnet/utils/visualization.py]: @jcwbacker Should this be `ax2.axis('off')`?

File: .gitignore (hunk 2 above)
Comment (jcwbacker) [inline .gitignore]: The line was, in fact, intentionally duplicated. 
> +/.ipynb*/

When running "git status" my ipython notebook backup folders keep showing up in the list of files. Jay thought adding the slash might help, but it didn't seem to make a difference, so if you deleted it, then that's fine

File: autocnet/utils/visualization.py (hunk 3 above)
Comment (jcwbacker) [inline autocnet/utils/visualization.py]: yes

**Comment by jcwbacker** [comment]
from_adjacency_file() was changed back to from_adjacency() since it now accepts a file or a dict

**Pull request merged**